packages/stomptuner/PKGBUILD

```diff
@@ -0,0 +1,60 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+_name=StompTuner
+pkgname=${_name,,}
+pkgver=0.5
+pkgrel=1
+pkgdesc='A multi-format instrument strobe tuner plugin and JACK application'
+arch=(aarch64 x86_64)
+url='https://github.com/brummer10/StompTuner'
+license=(GPL-3.0-or-later)
+depends=(cairo dbus fftw glibc gcc-libs libx11 libxcursor libxext libxrandr)
+makedepends=(git)
+optdepends=(
+  'jack: for using the stand-alone programs with JACK'
+  'clap-host: for loading the CLAP format plugins'
+  'lv2-host: for loading the LV2 format plugins'
+  'vst-host: for loading the VST2 format plugins'
+  'vst3-host: for loading the VST3 format plugins'
+)
+groups=(clap-plugins lv2-plugins pro-audio vst-plugins vst3-plugins)
+_dpf_commit='51edc8b6e3c636f29b5fca801bfb2551b0634b91'
+_pugl_commit='e33b2f6b0cea6d6263990aa9abe6a69fdfba5973'
+source=(
+  "${pkgname}-$pkgver.tar.gz::https://github.com/brummer10/$_name/archive/refs/tags/v$pkgver.tar.gz"
+  "DPF-$_dpf_commit.tar.gz::https://github.com/DISTRHO/DPF/archive/$_dpf_commit.tar.gz"
+  "pugl-$_pugl_commit.tar.gz::https://github.com/DISTRHO/pugl/archive/$_pugl_commit.tar.gz"
+)
```

Comment:
IMO we should just use a git source with submodules, see for example the `vcvrack` PKGBUILD or [Arch Wiki](https://wiki.archlinux.org/title/VCS_package_guidelines#Git_submodules). The stomptuner repo could then have `#tag=v$pkgver` and a sha256sum for verification, which is then not necessary for the submodule sources. This would make it easier to update in the future :)

Reply:
Done.

Reply:
just pushed a small fix, hope that's ok 🙏

Reply:
Yeah, good that you checked, I had a checksum there and then ran `updpkgsums` after I changed the Github URL and didn't check the `PKGBUILD` afterwards again. I thought that it now handles checksums for git repos, but apparently it doesn't.

Reply:
It does, just seems the extra `?` confused it :D